### PR TITLE
Adding SQFC to Antistasi

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    This speeds up start up time by eliminating a lot of script compiling.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Comment by killerswin2** [comment]
https://community.bistudio.com/wiki/SQF_Bytecode

**Comment by killerswin2** [comment]
Before:
Screenshot 2022-12-27 143958
After:
Screenshot 2022-12-27 163850

Also this will be faster If the UPSMON addon is merged, as those functions will be complied the same way.

**Comment by jaj22** [comment]
Having to update the sqfcs on every push sounds like too much of a pain (also github data spam, because they're binaries), and this should be reserved for releases if anything. What's the gain exactly?

**Comment by killerswin2** [comment]
Mainly for release, it saves a good amount of time because it will preprocess files, but the biggest benefit is that it does some compiler optimizations like constant folding and also bytecode compiling in engine is faster.

File: Tools/Builder/buildAddons.ps1 [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
+    "Copying files to temp dir"
+    New-Item -Path "temp" -ItemType Directory -Force > $null
+    Set-Location "$PSScriptRoot\..\.."
+    Copy-Item -Path "A3A" -Destination "temp" -Recurse -Force
+    
+    # we need to have temp directory as SymbolicLink 
+    New-Item -Type SymbolicLink -Path "P:\x\A3A" -Target "$PSScriptRoot\..\..\temp\A3A" -Force
+
+    "Compiling To ByteCode"
+    .\Tools\scriptCompiler\ArmaScriptCompiler.exe > scriptCompiler.sqfclog
```
Comment (Giddius) [inline Tools/Builder/buildAddons.ps1:30]: probably should make this absolute also with `$PSScriptRoot`

File: Tools/Builder/buildAddons.ps1 [hunk 2]
```diff
@@ -1,12 +1,43 @@
+#Requires -RunAsAdministrator
```
Comment (Giddius) [inline Tools/Builder/buildAddons.ps1:1]: adding a `#Requires -Version 6.0` or `4.0` would be good as some vars like `$PSScriptRoot`actually have only been implemented from some version on

**Review by Giddius — APPROVED** [review]

**Comment by Bob-Murphy** [comment]
Benched due to concerns

**Pull request closed without merging**